src/NzbDrone.Core/Datastore/Migration/201_add_bluray576_quality_in_profile.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using FluentMigrator;
+using Newtonsoft.Json;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(201)]
+    public class add_bluray576_quality_in_profile : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.WithConnection(ConvertProfile);
+        }
+
+        private void ConvertProfile(IDbConnection conn, IDbTransaction tran)
+        {
+            var updater = new ProfileUpdater201(conn, tran);
+
+            updater.SplitQualityAppend(13, 22);  // Insert Bluray576p after Bluray480p
+
+            updater.Commit();
+        }
+    }
+
+    public class Profile201
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public int Cutoff { get; set; }
+        public List<ProfileItem201> Items { get; set; }
+    }
+
+    public class ProfileItem201
+    {
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public int? Quality { get; set; }
+
+        public bool Allowed { get; set; }
+        public List<ProfileItem201> Items { get; set; }
+    }
+
+    public class ProfileUpdater201
+    {
+        private readonly IDbConnection _connection;
+        private readonly IDbTransaction _transaction;
+
+        private List<Profile201> _profiles;
+        private HashSet<Profile201> _changedProfiles = new HashSet<Profile201>();
+
+        public ProfileUpdater201(IDbConnection conn, IDbTransaction tran)
+        {
+            _connection = conn;
+            _transaction = tran;
+
+            _profiles = GetProfiles();
+        }
+
+        public void Commit()
+        {
+            foreach (var profile in _changedProfiles)
+            {
+                using (var updateProfileCmd = _connection.CreateCommand())
+                {
+                    updateProfileCmd.Transaction = _transaction;
+                    updateProfileCmd.CommandText = "UPDATE QualityProfiles SET Name = ?, Cutoff = ?, Items = ? WHERE Id = ?";
```

Comment:
These queries are invalid for Postgres, you'll need to mirror the existing migrations and correctly quote the columns/table names.